src/components/Auth/Login/index.js

```diff
@@ -1,32 +1,60 @@
 import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import { Container } from "../../Global/styled";
+import Loading from "../../Global/Loading";
+import TopBar from "../../Onboarding/TopBar";
+import LoginWithEmail from "./LoginWithEmail";
+import LoginOptions from "./LoginOptions";
+import { connect } from "react-redux";
 
-// import styled from "styled-components";
+class Login extends React.Component {
+  render() {
+    // sets path variable
+    const { path } = this.props.match;
 
-// import { Button, Form, Input, ButtonWrapper } from "../../Global/styled";
-// import { Linkton } from "../../Global/styled";
+    // once user logs in isLoggedIn will be true and route you to home page
+    const { isLoggedIn, loggingIn } = this.props;
+    if (isLoggedIn) return <Redirect to="/" />;
+    if (loggingIn) return <Loading />;
 
-const Login = props => {
-  return (
-    <h1>Login Page</h1>
-    // <SignInWrapper>
-    //   <h2>Sign in</h2>
-    //   <Form onSubmit={e => e.preventDefault() && false}>
-    //     <Input name="email" type="text" placeholder="Email" />
+    // Sets up routes for Login pages
+    return (
+      <Container justify="center" fluid={true}>
+        <TopBar {...this.props} />
 
-    //     <Input name="password" type="password" placeholder="Password" />
-    //     <ButtonWrapper>
-    //       <Button>Sign in</Button>
-    //       <Linkton to="/register">Register now!</Linkton>
-    //     </ButtonWrapper>
-    //   </Form>
-    // </SignInWrapper>
-  );
-};
+        <Route
+          exact
+          path={path}
+          render={props => (
+            <LoginOptions
+              {...props}
+              path={path}
+              handleGoogleAuth={this.handleGoogleAuth}
+              handleFacebookAuth={this.handleFacebookAuth}
+            />
+          )}
+        />
+        <Route
+          path={`${path}/email`}
+          render={props => (
+            <LoginWithEmail
+              {...props}
+              path={path}
+              handleLogin={this.handleLogin}
+            />
+          )}
+        />
+      </Container>
+    );
+  }
+}
 
-// const SignInWrapper = styled.div`
-//   display: flex;
-//   flex-direction: column;
-//   align-items: center;
-// `;
+const mapStateToProps = state => {
+  return {
+    // when user is not logged in isEmpty is true
+    isLoggedIn: !state.firebase.auth.isEmpty,
```

Comment:
for the `isEmpty` var, is there a more descriptive variable name we could use here?